frontends/web/src/containers/SubmitInterface.js

```diff
@@ -289,7 +290,15 @@ class SubmitInterface extends React.Component {
                                 className="submit-btn button-ellipse text-uppercase my-4"
                                 disabled={isSubmitting}
                               >
-                                Save
+                                {isSubmitting ? (
+                                  <Spinner
+                                    as="span"
+                                    animation="border"
+                                    size="sm"
+                                  />
+                                ) : (
+                                  "Add New Dataset"
```

Comment:
Can we change "Add New Dataset" here to "Upload Predictions"?

Reply:
sure, makes sense

Reply:
oh yes, my bad